platforms/hdp2.6.5/docs/HDP-installation.md

```diff
@@ -36,79 +47,146 @@ operating environment, and are called out in detail in the
 5. Open the url `your_node_url/ambari-repo` to confirm it can be accessed by yum.
-6. Put a file named `splicemachine.repo` under ``/etc/yums.repo.d/` with the content is as below
+6. On the ambari server node terminal, create a file named `splicemachine.repo` under ``/etc/yums.repo.d/` with the content as seen below, replacing "your_node_url" text with the hostname used in the test on step 5:
 
-  ````
-  [splicemachine]
-  name=SpliceMachine Repo
-  baseurl=http://your_node_url/ambari-repo
-  enabled=1
-  gpgcheck=0
-  ````
-7. Run `yum list | grep splicemachine` to make sure the custom repo is up and running.
+````
+[splicemachine]
+name=SpliceMachine Repo
+baseurl=http://your_node_url/ambari-repo
+enabled=1
+gpgcheck=0
+````
 
-Perform the following steps **on each node** in your cluster:
 
-Install the Splice Machine custom Ambari service rpm using the following command (take version 2.5.0.1811 for example) :
+7. Run `sudo yum list | grep splicemachine` to make sure the custom repo is up and running.
 
-```
-sudo yum install splicemachine_ambari_service
-```
+````
+$ sudo yum list | grep splicemachine
+splicemachine.noarch          hdp2.6.1.2.7.0.1915.p0.313-1
+                                                   splicemachine
+````
+
+8. Install the Splice Machine custom Ambari service rpm using the following command (take version for example) :
+
+````
+$ sudo rpm -ivh splicemachine_ambari_service-hdp2.6.1.2.7.0.1915.p0.313-1.noarch.rpm
+Preparing...                ########################################### [100%]
+   1:splicemachine_ambari_se########################################### [100%]
+````
+
+9. After install of the splicemachine_ambari_service rpm, restart ambari-server using `sudo service ambari-server restart` on the master node.
 
-After install the rpm, restart ambari-server using `service ambari-server restart`.
 
 
 ## Install splicemachine using Ambari service
 
 Follow the steps to install splicemachine server.
 
-1. Click the action button on the left bottom of the ambari page,then click on 'Add Services'
+0. On the Ambari Dashboard - Make a note of the current version of each service running on the cluster by looking at their individual config views.
+Note: These versions will be used for the backout procedure when uninstalling Splice Machine
+
+1. Click the "Action" button on the left bottom of the services list, select "Add Service"
 
 <img src="docs/add_services.jpg" alt="Add Service" width="400" height="200">
 
 2. Choose splice machine from the 'add service wizard'
 
 <img src="docs/add_service_wizard.jpg" alt="Add Service Wizard" width="400" height="200">
 
-3. Choose the master machine. It needs to be the same machine as the HBase master.
-
-4. Choose hosts needed to install splice machine. Only choose hosts that have hbase region server 
-installed.Then click next.
+3. Choose hosts needed to install splice machine. Choose both HBase master and HBase region servers. Then click next.
```

Comment:
Minor question here -should this cover configuring HBase for HA now, or is that somewhere else?

Reply:
It probably should, but I'm not the right person. I don't know how to do that.